unit_12/main.js

```diff
@@ -0,0 +1,64 @@
+// Exercise 1
+const compareArray = (array1, array2) => {
+  if (JSON.stringify(array1.sort()) === JSON.stringify(array2.sort())) {
+    return true
+  } else  {
+    return false
+  } 
+}
+// Exercise 2:
+const newArray = []
+const flattenArray = array => {
+  for (let i = 0; i < array.length; i++) {
+    if (Array.isArray(array[i])) {
+      flattenArray(array[i])
+    } else {
+      newArray.push(array[i])
+    }
+  }
+  return newArray
+}
+
+// Exercise 3:
+const chunksArray = (array, size) => {
+  const newArray = []
+  while (array.length) {
+    newArray.push(array.splice(0, size))
+  }
+  return newArray
+}
+
+// Exercise -4
+function ResultArray(array1, array2) {
```

Comment:
```suggestion
function resultArray(array1, array2) {
```

Đọc kỹ về `camel case`, tên hàm đặt kiểu gì trên kia ? xem quy tắc đặt tên hàm là loại từ gì